appveyor.yml

```diff
@@ -14,9 +12,8 @@ install:
   - ps: wget 'https://s3.amazonaws.com/kylo-pl-bucket/yaml-cpp-0.5.1-x86_64_mingw-w64_4.8.3_posix_seh.7z' -OutFile "$pwd\yaml-cpp.7z"
   - ps: 7z.exe x yaml-cpp.7z -oC:\tools | FIND /V "ing  "
 
-  - ps: (Get-Content C:\Ruby21-x64\lib\ruby\2.1.0\dl.rb) | Foreach-Object {$_ -replace "warn ", "puts "} | Set-Content C:\Ruby21-x64\lib\ruby\2.1.0\dl.rb
-  - ps: gem install bundler -q --no-ri --no-rdoc
-  - ps: bundle install --gemfile=lib/Gemfile --quiet 2>&1 | out-null
+  - gem install bundler --quiet --no-ri --no-rdoc
+  - bundle install --jobs 4 --retry 2 --gemfile=lib/Gemfile --quiet
```

Comment:
Although the one that comes with ruby 2.1.x should be sufficient not to require it.